packages/renderer-bootstrap4/renderers/table-linked-struct-field-renderer.tsx

```diff
@@ -31,18 +35,18 @@ const Bootstrap4TableLinkedStructFieldRenderer = ({
           <td>{x || ' '}</td>
         ))}
         <td>
-          <div className={createCssClass(cssName, 'row', 'actions')}>
+          <div className="text-nowrap " + {createCssClass(cssName, 'row', 'actions')}>
```

Comment:
What the significance of this? Is this something that absolutely needs to be in the library's renderer or is this specific to the application that uses it.

Also for concatenating class names use `combineCssClasses`. There are examples of it being used in other places.